<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FinanceCompta</class>
 <widget class="QWidget" name="FinanceCompta">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>506</width>
    <height>436</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Finance et comptabilité</string>
  </property>
  <property name="windowIcon">
   <iconset resource="ressource.qrc">
    <normaloff>:/db_test/Nouveau dossier/shopping-en-ligne.png</normaloff>:/db_test/Nouveau dossier/shopping-en-ligne.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::Shape::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Shadow::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Finance</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame_2">
     <property name="frameShape">
      <enum>QFrame::Shape::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Shadow::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QFrame" name="frame_6">
        <property name="frameShape">
         <enum>QFrame::Shape::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Shadow::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Du :</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDateEdit" name="dateDebut"/>
         </item>
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Au :</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDateEdit" name="dateFin"/>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Orientation::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="btnFiltrer">
           <property name="styleSheet">
            <string notr="true">QPushButton{
	border:none;
	image: url(:/Downloads/CartoBMW/img/systeme-de-filtrage-des-donnees.png);
}
QPushButton:hover{
	image: url(:/Downloads/CartoBMW/img/filtrage-des-donnees.png);
	
}</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnReinitialiser">
           <property name="styleSheet">
            <string notr="true">QPushButton{
	border:none;
	
	image: url(:/Downloads/CartoBMW/img/faire-pivoter-le-symbole-dinterface-fleche-circulaire-gauche.png);
}
QPushButton:hover{
	image: url(:/Downloads/CartoBMW/img/annuler.png);
	
}</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit">
        <property name="styleSheet">
         <string notr="true">QLineEdit{
	border: 1px solid white;
	border-radius: 5px;
}

QLineEdit:focus{
	border: 1px solid black;
}</string>
        </property>
        <property name="placeholderText">
         <string>Recherche...</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTabWidget" name="tabWidget">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="tab_3">
         <attribute name="title">
          <string>+</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QTableWidget" name="tableCompta">
            <column>
             <property name="text">
              <string>Identifiant</string>
             </property>
             <property name="font">
              <font>
               <bold>true</bold>
              </font>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Nom du produit</string>
             </property>
             <property name="font">
              <font>
               <bold>true</bold>
              </font>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Nom client</string>
             </property>
             <property name="font">
              <font>
               <bold>true</bold>
              </font>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Quantité vendu</string>
             </property>
             <property name="font">
              <font>
               <bold>true</bold>
              </font>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Date de la vente</string>
             </property>
             <property name="font">
              <font>
               <bold>true</bold>
              </font>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Prix de base</string>
             </property>
             <property name="font">
              <font>
               <bold>true</bold>
              </font>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Prix de vente</string>
             </property>
             <property name="font">
              <font>
               <bold>true</bold>
              </font>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Bénéfice</string>
             </property>
             <property name="font">
              <font>
               <bold>true</bold>
              </font>
             </property>
            </column>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="frame_5">
            <property name="frameShape">
             <enum>QFrame::Shape::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Shadow::Raised</enum>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <spacer name="horizontalSpacer">
               <property name="orientation">
                <enum>Qt::Orientation::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QLabel" name="label_2">
               <property name="font">
                <font>
                 <pointsize>12</pointsize>
                 <bold>true</bold>
                </font>
               </property>
               <property name="text">
                <string>Bénéfice de ce mois :</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="labelBenefice">
               <property name="font">
                <font>
                 <pointsize>12</pointsize>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">color: rgb(0, 170, 0);</string>
               </property>
               <property name="text">
                <string>0</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab_4">
         <attribute name="title">
          <string>-</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <widget class="QTableWidget" name="tableCharge">
            <column>
             <property name="text">
              <string>Identifiant</string>
             </property>
             <property name="font">
              <font>
               <bold>true</bold>
              </font>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Description</string>
             </property>
             <property name="font">
              <font>
               <bold>true</bold>
              </font>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Personne associé</string>
             </property>
             <property name="font">
              <font>
               <bold>true</bold>
              </font>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Prix</string>
             </property>
             <property name="font">
              <font>
               <bold>true</bold>
              </font>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Date</string>
             </property>
             <property name="font">
              <font>
               <bold>true</bold>
              </font>
             </property>
            </column>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="frame_4">
            <property name="frameShape">
             <enum>QFrame::Shape::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Shadow::Raised</enum>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QPushButton" name="btnAjouterCharge">
               <property name="text">
                <string>Ajouter</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_3">
               <property name="orientation">
                <enum>Qt::Orientation::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QLabel" name="label_5">
               <property name="font">
                <font>
                 <pointsize>12</pointsize>
                 <bold>true</bold>
                </font>
               </property>
               <property name="text">
                <string>Charge de ce mois :</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_charge">
               <property name="font">
                <font>
                 <pointsize>12</pointsize>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">color: rgb(170, 0, 0);</string>
               </property>
               <property name="text">
                <string>0</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frame_3">
        <property name="frameShape">
         <enum>QFrame::Shape::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Shadow::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="label_6">
           <property name="font">
            <font>
             <pointsize>12</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Bénefices avec charges: </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="labelBeneficeCharge">
           <property name="font">
            <font>
             <pointsize>12</pointsize>
             <bold>false</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(0, 85, 255);</string>
           </property>
           <property name="text">
            <string>0</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_4">
           <property name="orientation">
            <enum>Qt::Orientation::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="ressource.qrc"/>
 </resources>
 <connections/>
</ui>
